components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/ApisApi.java

```diff
@@ -275,5 +275,51 @@ public Response apisApiIdDocumentsDocumentIdDelete(@ApiParam(value = "**API ID**
     throws NotFoundException {
     return delegate.apisApiIdDocumentsDocumentIdDelete(apiId,documentId,ifMatch,ifUnmodifiedSince);
     }
+    @GET
```

Comment:
Please note that these files get auto-generated using the swagger-to-cxf maven plugin we use for REST API development.